go.mod

```diff
@@ -15,6 +15,7 @@ require (
 	github.com/guelfey/go.dbus v0.0.0-20131113121618-f6a3a2366cc3 // indirect
 	github.com/hinshun/vt10x v0.0.0-20180809195222-d55458df857c // indirect
 	github.com/jinzhu/copier v0.0.0-20180308034124-7e38e58719c3
+	github.com/karalabe/xgo v0.0.0-20190301120235-2d6d1848fb02 // indirect
```

Comment:
dont think we need this dep, it is only for our cross compile, not sure how to exclude it from the go.mod

Reply:
i've removed this as part of the clean-up rebase.